jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -60,4 +68,11 @@ then
   sudo chown -R ${USER}:${USER} "${LOGS_DIR}/upgrade"
 fi
 
+if [[ "${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_centos" ]]
```

Comment:
by doing this, you discard all logs that were on the management cluster, so for example if the run fails before pivoting you will have no logs at all.

Reply:
I am already collecting the logs pre-pivot on line 49, would you please explain the scenario a bit more.

Reply:
oh, I didn't realize that you changed the directory in different cases. sorry for that, I probably reviewed too fast. Could maybe instead of pre_pivot and post_pivot use "mgmt" and "target" ?

Reply:
Done, thanks